shopfloor/services/checkout.py

```diff
@@ -85,17 +85,7 @@ def scan_document(self, barcode):
         if not picking:
             package = search.package_from_scan(barcode)
             if package:
-                pickings = (
-                    self.env["stock.move.line"]
-                    .search(
-                        [
-                            ("state", "not in", ("cancel", "done")),
-                            ("package_id", "=", package.id),
-                            ("picking_id.picking_type_id", "=", self.picking_type.id),
-                        ]
-                    )
-                    .mapped("picking_id")
-                )
+                pickings = search.stock_picking_from_package(package, self.picking_type)
```

Comment:
I modified this here: https://github.com/camptocamp/wms/pull/8/commits/92a58e6ea608e0354b7adc1a7be568f38f1ec924

I'll take care of it in my rebase.